cmd/kflex/ctx/ctx.go

```diff
@@ -64,39 +67,46 @@ func (c *CPCtx) Context(chattyStatus, failIfNone bool) {
 		} else if failIfNone {
 			if !c.isCurrentContextHostingClusterContext() {
 				fmt.Fprintln(os.Stderr, "The hosting cluster context is not known!\n"+
-					"You can make it known to kflex by doing `kubectl config use-context $name_of_hosting_context` "+
-					"and then either `kflex ctx` or `kflex create ...`")
+					"You can make it known to kflex by doing `kubectl ctx --force-replace-for-hosting $name_of_hosting_context` "+
```

Comment:
(0) This PR should include a doc update.

(1) Note that Issue #288 calls out two _different_ issues. This PR addresses the one labeled (B), in two ways. I think that the more succinct one would be sufficient. The wordier one, `kflex ctx --force-replace-for-hosting $name_of_hosting_context`, is inconsistent with the normal syntax of `kflex ctx $control_plane_name`.

Reply:
@MikeSpreitzer thanks, so is the suggestion then that only `--set-current-for-hosting` is sufficient?  I am not sure I understood then the original suggestion in #288 (A)  then "There should be something like kflex ctx --force $something that will replace the context even if it already exists." - could you pls. elaborate more on what your suggestion for (A) is?

Reply:
Item (B) in Issue 288 concerns what happens when my kubeconfig file contains a context named "foo" but that is outdated or otherwise incorrect. I want to be able to make `kflex ctx foo` _set_ context "foo" to be the right stuff for the control plane "foo" regardless of whether I already had a context named "foo".

Reply:
This is so that I can simplify our instructions by replacing

```shell
kubectl config delete-context wds1 || true
kflex ctx wds1
```

with

```shell
kflex ctx --ignore-existing-context wds1
```
(or whatever is a good name for the flag)

Reply:
I see, thanks, so is this about replacing the all the elements associated with a context (context name, clusters, auth) ?

Reply:
Right. The idea is to not assume that just because there is a context with the right name that means that it has the right contents of any form. It might be quite old and outdated.